monero/client_test.go

```diff
@@ -51,13 +49,10 @@ func TestClient_Transfer(t *testing.T) {
 	kpABPub := mcrypto.SumSpendAndViewKeys(kpA.PublicKeyPair(), kpB.PublicKeyPair())
 	vkABPriv := mcrypto.SumPrivateViewKeys(kpA.ViewKey(), kpB.ViewKey())
 
-	r, err := rand.Int(rand.Reader, big.NewInt(10000))
-	require.NoError(t, err)
-
 	cXMRTaker := NewClient(common.DefaultXMRTakerMoneroEndpoint)
 
 	// generate view-only account for A+B
-	walletFP := fmt.Sprintf("test-wallet-%d", r)
```

Comment:
@noot What does the `FP` in `walletFP` stand for?

Reply:
filepath!